### PR TITLE
Renamed some short arguments

### PR DESCRIPTION
**Pull request by propensive**
Both files and directories are now specified with `-f`, which frees up `-d` to be used for dependencies.

### CODE REVIEW

File: src/build/build.scala [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /*
    ╔═══════════════════════════════════════════════════════════════════════════════════════════════════════════╗
-   ║ Fury, version 0.7.14. Copyright 2018-19 Jon Pretty, Propensive OÜ.                                         ║
```
Comment (odisseus) [inline src/build/build.scala]: Could you separate this change into another pull request?

File: src/core/args.scala [hunk 2]
```diff
@@ -59,9 +59,9 @@ object Args {
   val CompilerArg = CliParam[String]('c', 'compiler, "specify a compiler")
   val ClassArg = CliParam[String]('C', 'class, "specify a class name")
   val DefaultCompilerArg = CliParam[String]('c', 'compiler, "specify a default compiler")
-  val LinkArg = CliParam[String]('l', 'link, "specify a dependency link to another module")
-  val SourceArg = CliParam[String]('d', 'source, "specify a source directory")
-  val DirArg = CliParam[Path]('d', 'dir, "specify the new repository destination directory")
+  val DependencyArg = CliParam[String]('d', 'dependency, "specify a dependency on another module")
+  val SourceArg = CliParam[String]('f', 'source, "specify a source directory")
+  val DirArg = CliParam[Path]('f', 'dir, "specify the new repository destination directory")
```
Comment (odisseus) [inline src/core/args.scala:62]: I think the previous flags were better. There is a difference between adding a source _file_ (which Fury does not support) and adding a source _directory_ that may or may not contain files at the moment.

Comment (propensive) [reply src/core/args.scala:62]: I would like that, but I also want to have the invariant that for any entity type, the parameter corresponding to it should be a lower-case letter which is the first letter of the type of entity, where the reference is being made to an entity in the current layer. I would also like to use the same letter, but in upper-case, to refer to the same type of entity, but when it's external to the current layer.

Comment (propensive) [reply src/core/args.scala:62]: We currently can't use `-s` for sources because `-s` is for schemas. I'm considering renaming "schemas" to "variants" and thus changing its parameter to `-v`, though. See #759.

Comment (odisseus) [reply src/core/args.scala:62]: I'm not sure I understand your intent completely, but I support any kind of system which would help memorizing the arguments.

**Review by odisseus — APPROVED** [review]

**Comment by odisseus** [comment]
The command `layer extract` accepts both `FileArg` and `DirArg`. Now that they both have the flag `-f`, that command is broken.

**Review by odisseus — CHANGES_REQUESTED** [review]
The command `layer extract` [accepts](https://github.com/propensive/fury/blob/master/src/build/build.scala#L511) both `FileArg` and `DirArg`. Now that they both have the flag `-f`, that command is broken.
